plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_remote_invoke.py

```diff
@@ -33,11 +33,11 @@ def launch_jobs(temp_dir: str) -> None:
         singleton_state = job_spec["singleton_state"]
         sweep_configs = job_spec["sweep_configs"]
         task_function = job_spec["task_function"]
-
+        ray_init_cfg = job_spec["ray_init_cfg"]
         instance_id = _get_instance_id()
-
         sweep_dir = None
 
+        start_ray(ray_init_cfg)
```

Comment:
`ray.init` only needs to be called once, not every time we call `run_job.remote`.

Reply:
See previous comment, I don't think we need to pickle an extra field for the purpose of this. I guess if you want to avoid call `start_ray` in each job you can get the `launcher.ray.init` config in the loop and only call `start_ray` once, however I'm not sure if that's much cleaner.